get_github_coderabbit_review/tests/integration/test_e2e_workflow.py

```
@@ -0,0 +1,518 @@
+    def setUp(self):
+        """Set up test fixtures."""
+        self.temp_dir = tempfile.mkdtemp(prefix="uvx_test_")
+    
+    def tearDown(self):
+        """Clean up test fixtures."""
+        if os.path.exists(self.temp_dir):
+            for file in os.listdir(self.temp_dir):
+                os.unlink(os.path.join(self.temp_dir, file))
+            os.rmdir(self.temp_dir)
+    
+    @unittest.skipIf(os.system("which uvx") != 0, "uvx not available")
```

Ruff (0.13.1)

Starting a process with a shell: seems safe, but may be changed in the future; consider rewriting without `shell`

(S605)